app/views/admin/link_check_reports/_link_check_report.html.erb

```diff
@@ -17,9 +17,17 @@
     <section class="alert alert-warning">
       <h3 class="remove-top-margin">Broken links</h3>
```

Comment:
Could we group the links into broken links and caution links? And show the broken links at the top?

Reply:
I don't think we differentiate between broken and caution links?

Reply:
The link reports get a status of `broken` or `caution`. - https://github.com/alphagov/link-checker-api/blob/master/docs/api.md#linkreport-entity